ui/src/main/js/view/templates/stage-logs.hbs

```diff
@@ -5,6 +5,9 @@
         <div class="node-log-frame {{this.status}}" cbwf-controller="node-log" objectUrl="{{this._links.log.href}}">
             <div class="node-name"><span class="glyphicon glyphicon-collapse-down" title="Expand"></span><span class="glyphicon glyphicon-collapse-up" title="Collapse"></span>
                 {{this.name}}  {{#if this.durationMillis}}(self time {{formatTime this.durationMillis}}){{/if}}
+                {{#if this.parameterDescription}}
+                    - {{this.parameterDescription}}
```

Comment:
You're right, good point.  I switched to

"name -- argumentsString -- (self time 10 ms)"
I.e. 
"Shell Step -- rm -rf / -- (self time 10 hours)"